onnx/defs/math/defs.cc

```diff
@@ -536,18 +536,21 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input and output types to signed numeric tensors.")
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
-static const char* LeakyRelu_ver6_doc = R"DOC(
+static const char* LeakyRelu_ver16_doc = R"DOC(
 LeakyRelu takes input data (Tensor<T>) and an argument alpha, and produces one
 output data (Tensor<T>) where the function `f(x) = alpha * x for x < 0`,
 `f(x) = x for x >= 0`, is applied to the data tensor elementwise.
+
+**History**
+- Version 16 adds bfloat16 to the types allowed.
```

Comment:
I like providing a history section, which makes the change clearer. Just out of curiosity: Why only LeakyRelu and PRelu add history section?

Reply:
Good question! For the control-flow ops, especially, I was thinking about another issue: they have very long documentation, and we end up copy-pasting it when we update the op even though the documentation doesn't change much, which also presumably increases binary size etc. I was thinking a slightly better solution might be needed, if we wanted to avoid this kind of increase. Something to think about. Thanks!

Reply:
Oh my goodness, a history section in the actual op would be so useful. I spend so much time digging through dozens of PR's each ONNX update and comparing Changelog.md entries to see what actually changed between ops for a given version.

Reply:
@fdwr : I agree it is cumbersome to find out op updates. In short term you can refer to the release wiki pages that we create to track op validation. This one is from last release :  https://github.com/onnx/onnx/wiki/Logistics-for-ONNX-Release-1.10#changelog
We will create a similar one for 1.11 soon. This table has a short description on what change and link to onnx PR which made the update.

Reply:
@askhade : 👍 That wiki looks nice. I read the generic notes on the Release page, but the wiki has more notes beyond just the PR title and a link.